src/cli/index.js

```diff
@@ -142,7 +142,7 @@ if (outputWrapper) {
 if (command.noArguments && commander.args.length) {
   reporter.error(reporter.lang('noArguments'));
   reporter.info(command.getDocsInfo);
-  process.exit(1);
+  process.exitCode = 1;
```

Comment:
You need to shortcut the rest of the code to have the same behavior - `process.exit` doesn't return, but setting `process.exitCode` will continue executing the rest of the code.